### PR TITLE
pqsql: Fixed timeout issue and enable Azure support for scale

### PR DESCRIPTION
**Pull request by tiffanyn108**
Fixed https://github.com/red-hat-storage/ocs-ci/issues/3455
Enable Azure support.

Signed-off-by: Tiffany Nguyen <tunguyen@redhat.com>

### CODE REVIEW

**Comment by tiffanyn108** [comment]
For some reason, Jenkins report is not correct.
Test case is passed as shown in the log: https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/16079/testReport/

File: ocs_ci/ocs/scale_pgsql.py [hunk 1]
```diff
@@ -148,21 +148,10 @@ def add_worker_node(instance_type=None):
...
-        and config.ENV_DATA["platform"].lower() == "baremetal"
-    ):
-        log.info("Running scale test on existing worker nodes.")
-    elif (
-        config.ENV_DATA["deployment_type"] == "upi"
-        and config.ENV_DATA["platform"].lower() == "azure"
-    ):
-        raise UnsupportedPlatformError("Unsupported Platform")
+    elif config.ENV_DATA["deployment_type"] == "upi" and [
+        config.ENV_DATA["platform"].lower() == "vsphere" or "baremetal" or "azure"
```
Comment (keesturam) [inline ocs_ci/ocs/scale_pgsql.py:152]: azure is 'ipi'

File: ocs_ci/ocs/scale_pgsql.py [hunk 2]
```diff
@@ -106,14 +105,12 @@ def add_worker_node(instance_type=None):
...
+
         # Create machineset for app worker nodes on each zone
         for obj in machine.get_machineset_objs():
             if "app" in obj.name:
                 ms_name.append(obj.name)
-        if instance_type is not None:
-            instance_type = instance_type
-        else:
-            instance_type = "m5.4xlarge"
+        instance_type = instance_type if instance_type else "m5.4xlarge"
```
[the remark this answers is not in the document]
Comment (tiffanyn108) [reply ocs_ci/ocs/scale_pgsql.py:113]: I will look into this change for Azure.

[the remark this answers is not in the document]
Comment (ramkiperiy) [reply ocs_ci/ocs/scale_pgsql.py:113]: @tiffanyn108 please re-work on this PR #4467 (RHV) #4150 (Azure), we have now support for scaling nodes in Azure and RHV using custom machineset.

**Review by ramkiperiy — CHANGES_REQUESTED** [review]
This PR needs rebase with Azure and RHV support PR's merged recently.

**Comment by tiffanyn108** [comment]
Closing this PR.  Will need to revisit later.

**Pull request closed without merging**